int32 armPos
int32 bucketPos
int32 armVel
int32 bucketVel
